### PR TITLE
[CP-2754] [API Device][Kompakt] About your Kompakt

### PR DESCRIPTION
**Pull request by patrycja-paczkowska**
JIRA Reference: [CP-2754](https://appnroll.atlassian.net/browse/CP-2754)

### :memo: Description ️

-

### :muscle: Checklist before requesting a review - nice to have

- getState function in async thunk actions is correctly typed
- redux selectors are used in components / prop drilling is reduce

### :exclamation: Checklist before merging a pull request

- change went through the QA process
- translations are updated in dedicated application
- [CHANGELOG.md](./CHANGELOG.md) is updated

[CP-2754]: https://appnroll.atlassian.net/browse/CP-2754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/mudita-center-e2e/src/page-objects/overview.page.ts [hunk 1]
```diff
@@ -3,10 +3,13 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 export class OverviewPage extends Page {
+  public get overviewMenuItem() {
```
Comment (lkowalczyk87) [inline apps/mudita-center-e2e/src/page-objects/overview.page.ts:9]: All elements related to left navigation menu are common for all views and may be moved to separate object

File: apps/mudita-center-e2e/src/specs/overview/kompakt-about.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  it("Verify About Your Device page", async () => {
+    // Subtitle
+    const aboutSubtitle = await aboutKompaktPage.aboutSubtitle
+    await expect(aboutSubtitle).toHaveText("Device details")
+    // SN
+    const serialNumberLabel = await aboutKompaktPage.serialNumberLabel
+    await expect(serialNumberLabel).toHaveText("Serial number")
+    const serialNumberValue = await aboutKompaktPage.serialNumberValue
+    const serial = await serialNumberValue.getText()
+    await expect(serialNumberValue).toHaveText(kompaktSerialNumberRegex)
```
Comment (lkowalczyk87) [inline apps/mudita-center-e2e/src/specs/overview/kompakt-about.ts:70]: `await expect(serialNumberValue).toHaveText(
      overviewDataWithOneSimCard.summary.about.serialNumber.text
    )`

File: apps/mudita-center-e2e/src/page-objects/overview.page.ts (hunk 1 above)
Comment (patrycja-paczkowska) [reply apps/mudita-center-e2e/src/page-objects/overview.page.ts]: menu.page added :)

File: apps/mudita-center-e2e/src/specs/overview/kompakt-about.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    await expect(sarButton).toBeClickable()
+  })
+
+  it("Go to SAR information", async () => {
+    const sarButton = await aboutKompaktPage.sarButton
+    await sarButton.waitForClickable()
+    await sarButton.click()
+    const sarHeader = await aboutKompaktPage.sarHeader
+    await expect(sarHeader).toHaveText("SAR")
+    // TBD: SAR content + scroll
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/overview/kompakt-about.ts:94]:
```suggestion
    //TODO SAR content + scroll
```

This way Visual Studio Code will recognize a comment as a To-Do item

**Review by tomaszmaleckimudita — APPROVED** [review]

**Pull request merged**